pkg/inst-api-semconv/instrumenter/utils/metrics_shadower.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func Shadow(attrs []attribute.KeyValue, metricsSemConv map[attribute.Key]bool) (int, []attribute.KeyValue) {
```

Comment:
Could you explain why we need to do this?

Reply:
User may need to customize the attributes in their own distribution, so we may need some extensions for this.